### ISSUE TITLE
[med] Color Contrast: Many Pages: Blue text over white background fails contrast ratio

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5467]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Press Tab key to focus the 'terms of service' link and press Enter key.
3. Measure the color contrast ratio of any blue link against white background.

## Expected Result:
The contrast ratio of the text against the background should be at least 4.50:1 for small text and 3.00:1 for large text (14pt bold or 18 pt+).

## Actual Result:
The blue links (#0185FF) against white background (#FFFFFF) have a color contrast ratio of 3.62:1.

HTML:
`<a href="https://www.expensify.com/privacy"><span style="text-decoration:underline">Privacy Policy</span></a>`

CSS:
`a {
color: #0185ff;
text-decoration: none;
}`

Other occurrences:
1. Licenses -> the blue links (#0185FF) against white background (#FFFFFF), have a color contrast ratio of 3.62:1.
2. Main chat section

## Workaround:
No workaround

## Area issue was found in:
Terms of service page, Licenses page, Chat page

## Failed WCAG checkpoints
1.4.3

## User impact:
Some users won't be able to read the text.

## Suggested resolution:
Use a darker shade of blue for text such as `#0074E0`, which will give a color contrast ratio of 4.59:1 against the white background.

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5180789_color_contrast_-_privacy_policy_-_terms_of_service](https://user-images.githubusercontent.com/88733897/134570820-e8227c62-dcc0-4cc0-a890-3fd120183246.png)
![Bug5180789_color_contrast_-_licenses](https://user-images.githubusercontent.com/88733897/134570839-d4f17a3b-3a3f-4b3c-9f6a-695932b7753a.png)
![Color_Contrast-Chat-Blue_link_text_on_white_fails_contrast](https://user-images.githubusercontent.com/88733897/134570848-342b6c28-fe79-407a-bac4-782c04331c5a.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shawnborton** [comment]
So I think I am seeing a pattern here where our brand colors (blue, green) aren't always accessible when used in the UI. So yeah, in the spirit of accessibility, I think it makes sense to have a slightly darker version of blue and green used for various components?

**Comment by shawnborton** [comment]
@ogumen when I am testing these myself, which of the following needs to be a pass? What's the difference between `WCAG AAA` and `WCAG AA`?
![image](https://user-images.githubusercontent.com/2319350/134697377-666c27a4-317e-45db-979d-feee654d84dd.png)

**Comment by ogumen** [comment]
@shawnborton the AA and AAA are different levels of WCAG conformance. The acceptable compliance is WCAG level AA, so this should be met
